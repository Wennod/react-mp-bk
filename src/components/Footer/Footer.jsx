import React from 'react';

// import Logo from '../LogoComponent/Logo';
import Style from './Footer.style.scss';

const Footer = (props) => {
    return (
        <footer className="footer">
            <img className="footer__logo" src="http://via.placeholder.com/100x25" alt="logo"/>                    
        </footer>
    );
} 

export default Footer;