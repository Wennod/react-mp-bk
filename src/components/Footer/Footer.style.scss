.footer {
    flex-shrink: 0;
    background-color: darkgrey;
}