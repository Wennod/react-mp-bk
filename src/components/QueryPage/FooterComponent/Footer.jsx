import React from 'react';

import Logo from '../LogoComponent/Logo';
import Style from './Footer.style.scss';

const Footer = (props) => {
    return (
        <footer className="footer">
            <Logo />            
        </footer>
    );
} 

export default Footer;