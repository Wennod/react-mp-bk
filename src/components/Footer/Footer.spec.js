import React from 'react';
import Footer from './Footer';
import renderer from 'react-test-renderer';

describe('Footer snapshot', () => {
    it('should match snapshot', () => {
        const tree = renderer
            .create(<Footer />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});