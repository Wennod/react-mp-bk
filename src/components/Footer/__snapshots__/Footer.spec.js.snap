// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot should match snapshot 1`] = `
<footer
  className="footer"
>
  <img
    alt="logo"
    className="footer__logo"
    src="http://via.placeholder.com/100x25"
  />
</footer>
`;
